addons/l10n_dk_oioubl/models/account_edi_xml_oioubl_201.py

```diff
@@ -211,9 +212,22 @@ def _get_invoice_monetary_total_vals(self, invoice, taxes_vals, line_extension_a
         return vals
 
     def _get_invoice_payment_terms_vals_list(self, invoice):
-        # cleaned atm because it's not mandatory and the standard payment terms gets the document rejected for validation
-        # https://www.oioubl.info/Classes/en/PaymentTerms.html
-        return []
+        # OVERRIDES account.edi.xml.ubl_20
```

Comment:
```suggestion
        # EXTENDS 'account_edi_ubl_cii'
```

Reply:
In that case, it's an override. I will change for the module.